Fix/separate tests

sunnytownsend opened the pull request.

sunnytownsend committed:
move app.R to root dir

- code did not run because paths to shape files changed
- move app.R to root dir
- move shape files so code now runs

sunnytownsend committed:
move tests to own script, call from app.R

sunnytownsend committed:
tidying up whitespace and #s

ONS-Crooma reviewed (approved):
All looks good to me! :shipit:

seb231 commented on test_model.R:
```diff
@@ -0,0 +1,40 @@
+test_that("Test that the number of wards in demand set is the number in GM (=215)",
```
Think this needs `library(testthat)`, no?

seb231 replied:
ah... nope wrong, sorry

seb231 commented:
Why have the .R files been moved to the root @sunnytownsend ?

sunnytownsend committed:
move & split out app, model fns & tests

- moved files so now app and tests run
- created run_tests.R file
- added lots of comments around testing
- created model_functions.R to hold all model fns and just source them
in both tests and app
